bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/UpnpDiscoveryService.java

```diff
@@ -102,6 +103,7 @@ protected void startScan() {
         }
         upnpService.getRegistry().addListener(this);
         upnpService.getControlPoint().search();
+        upnpService.getControlPoint().search(new RootDeviceHeader());
```

Comment:
I think we discussed that a while ago already: I would not want this (WeMo specific) call to be in the general UPnP discovery - I am pretty sure that it increases the network traffic quite a bit, which is not needed for any user that does not use WeMo devices.
I'd therefore suggest to move this call to a Wemo DiscoveryService, which does nothing else than triggering this (and the UPnPDiscoveryParticipant will then receive the results).